package-lock.json

```diff
@@ -1,612 +1,8 @@
 {
   "name": "vue-wordle",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Edaa can you remove package.json files from this PR. ഇതിന്റെ ആവശ്യമില്ലല്ലോ. Thanks

Reply:
oops... gitignore il ath illarnalle..ippa chyam

Reply:
aaand done

Reply:
@subins2000 achara prakaram LGTM :) idendath aanu

Reply:
ഇങ്ങനെ പറ്റൂലാ. Do this:
```
git checkout package.json package-lock.json
git commit -a
git push
```
ഈ PR ഇപ്പോൾ merge ചെയ്താൽ മൊത്തം അടിച്ച് പോകും cause you just removed the 2 core files in this PR 😬

Reply:
njn polum ariyathe njn rand file remove akkyo